draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -3456,808 +3548,1280 @@ ObjectLists MAY combine regular objects and secondary ObjectLists in the same ob
+Content-Length: 0
+Expires: Sun, 27 Oct 2024 08:49:21 GMT
+Server: example-server/0.1
+ETag: "4331492443626270781"
+Last-Modified: Sun, 27 Oct 2024 08:48:17 GMT
+Cache-Control: max-age=60
+Date: Sun, 27 Oct 2024 08:48:23 GMT
+Content-Type: application/cdni; ptype=ci-trigger-collection.v2
+
+REQUEST:
+
+GET /cit/3f2d259d-a980-4742-beeb-9392a58129f5 HTTP/1.1
+User-Agent: example-user-agent/0.1
+Host: dcdn.example
+Accept: */*
+If-None-Match: "6990548174277557683"
+If-Modified-Since: Sun, 27 Oct 2024 08:49:10 GMT
+
+RESPONSE:
+
+HTTP/1.1 304 Not Modified
+Content-Length: 0
+Expires: Sun, 27 Oct 2024 08:49:24 GMT
+Server: example-server/0.1
+ETag: "6990548174277557683"
+Last-Modified: Sun, 27 Oct 2024 08:48:17 GMT
+Cache-Control: max-age=60
+Date: Sun, 27 Oct 2024 08:48:24 GMT
+Content-Type: application/cdni; ptype=ci-trigger.v2
+
 ]]></artwork>
    </figure>
-   </section>
+   <t>
+   When the trigger processing is complete, the contents of the filtered collections will be updated.
+   dCDN SHOULD also update the "ETag" and/or "Last-Modified" response headers - whichever was previously sent - 
+   when delivering the updated collection representations.
+   dCDN SHOULD also use cache control headers, like "Expires" and "Cache-Control", to indicate how caching of
+   the resource representation should happen by uCDN and intermediate proxies. 
 
-   <section title="Polling for Changes in Status" anchor="examples-polling-status-changes"><t>
-   uCDN SHOULD use the ETags of collections or triggers
-   when polling for changes in status, as shown in the
-   following examples:</t>
+   For example, when the two example triggers are complete, the
+   collections of pending and complete triggers look as follows:</t>
 
    <figure><artwork><![CDATA[
 REQUEST:
 
-  GET /triggers/pending HTTP/1.1
-  User-Agent: example-user-agent/0.1
-  Host: dcdn.example.com
-  Accept: */*
-  If-None-Match: "4331492443626270781"
+GET /cit/state/pending HTTP/1.1
+User-Agent: example-user-agent/0.1
+Host: dcdn.example
+Accept: */*
+
+RESPONSE:
+
+HTTP/1.1 200 OK
+Content-Length: 51
+Expires: Sun, 27 Oct 2024 08:49:25 GMT
+Server: example-server/0.1
+ETag: "1337503181677633762"
+Last-Modified: Sun, 27 Oct 2024 08:48:17 GMT
+Cache-Control: max-age=60
+Date: Sun, 27 Oct 2024 08:48:25 GMT
+Content-Type: application/cdni; ptype=ci-trigger-collection.v2
+
+{
+  "staleresourcetime": 86400,
+  "triggers": []
+}
+
+REQUEST:
+
+GET /cit/state/complete HTTP/1.1
+User-Agent: example-user-agent/0.1
+Host: dcdn.example
+Accept: */*
+
+RESPONSE:
+
+HTTP/1.1 200 OK
+Content-Length: 193
+Expires: Sun, 27 Oct 2024 08:49:26 GMT
+Server: example-server/0.1
+ETag: "4481489539378529796"
+Last-Modified: Sun, 27 Oct 2024 08:48:17 GMT
+Cache-Control: max-age=60
+Date: Sun, 27 Oct 2024 08:48:26 GMT
+Content-Type: application/cdni; ptype=ci-trigger-collection.v2
+
+{
+  "staleresourcetime": 86400,
+  "triggers": [
+    "https://dcdn.example/cit/3f2d259d-a980-4742-beeb-9392a58129f5",
```

Comment:
Again, we might want to consider adding the invalidate example trigger URI to this list.

Reply:
fixed